src/corelibs/U2Formats/src/GTFFormat.cpp

```diff
@@ -360,6 +360,11 @@ bool parseAttributes(QString attributesStr, QMap<QString, QString>& parsedAttrVa
     int pos = 0;
 
     while (pos < attributesStr.size()) {
+        //skip sequence of spaces
+        while (attributesStr.indexOf(' ', pos) == pos) {
+            pos++;
+        }
```

Comment:
This code will skip all spaces, but the next code is supposed to find one space.

Reply:
Here we "rewind" to first non-space symbol, because string could be beginning with more thn one space.
Also this code made more simple.